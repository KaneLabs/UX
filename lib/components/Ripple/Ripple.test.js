var _interopRequireDefault=require("@babel/runtime/helpers/interopRequireDefault");var _react=_interopRequireDefault(require("react"));var _reactTestRenderer=_interopRequireDefault(require("react-test-renderer"));var _Ripple=_interopRequireDefault(require("./Ripple"));var _this=this,_jsxFileName="/Users/kane/Lab/Eros/src/components/Ripple/Ripple.test.js";test('Ripple Renders',function(){var tree=_reactTestRenderer.default.create(_react.default.createElement(_Ripple.default,{__self:_this,__source:{fileName:_jsxFileName,lineNumber:6,columnNumber:32}})).toJSON();expect(tree).toMatchSnapshot();});