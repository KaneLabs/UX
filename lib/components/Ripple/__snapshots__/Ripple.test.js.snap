// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ripple Renders 1`] = `
<div
  className="css-view-1dbjc4n"
  data-focusable={true}
  data-testid="mb-ripple"
  onClick={[Function]}
  onContextMenu={[Function]}
  onKeyDown={[Function]}
  tabIndex="0"
>
  <div
    className="css-view-1dbjc4n r-pointerEvents-633pao"
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.00)",
        "borderBottomLeftRadius": "0px",
        "borderBottomRightRadius": "0px",
        "borderTopLeftRadius": "0px",
        "borderTopRightRadius": "0px",
        "bottom": "0px",
        "left": "0px",
        "overflowX": "hidden",
        "overflowY": "hidden",
        "position": "absolute",
        "right": "0px",
        "top": "0px",
        "zIndex": 10,
      }
    }
  />
</div>
`;
